universum/configuration_support.py

```diff
@@ -8,14 +8,133 @@
     "Step",
     "Configuration",
     "Variations",
+    "combine",
     "set_project_root",
     "get_project_root"
 ]
 
 
+# pylint: disable-msg=line-too-long
 class Step:
     """
-    Step is a collection of configurable project-specific data needed for Universum operations.
+    Step is a collection of configurable project-specific data entries needed for Universum functionality. Individual
```

Comment:
Can we describe it in a more... plain way? Like, 'Step is a class to describe all parameters, passed to a single Universum launch' _(it's a bad description, it's just an example)._
You may leave this sentence somewhere as well, I'm just afraid, being totally correct, it still is more confusing than clarifying.